.drone.yml

```diff
@@ -0,0 +1,48 @@
+kind: pipeline
+name: amd
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: build
+  image: ocaml/opam2:4.07
+  commands:
+  - sudo apt-get update && sudo apt-get -y install afl
+  - sudo chown -R opam .
+  - git -C /home/opam/opam-repository pull origin && opam update
+  - opam switch 4.07+afl
+  - opam pin add -n .
+  - opam depext cstruct-async cstruct-lwt cstruct-unix cstruct ppx_cstruct
+  - opam install -y .
+  - opam install -y crowbar fmt
+  - opam exec -- dune build fuzz/fuzz.exe
+  - mkdir out && mkdir in && echo bactrian > in/test
```

Comment:
If you're using `bun` you don't need to do this anymore - `bun` should take care of it for you.

Reply:
i cant think of a pun involving bun but i pushed the change for fun and now i have to run